test/functional/pdo_sqlsrv/pdo_bigint_outparam.phpt

```diff
@@ -0,0 +1,82 @@
+--TEST--
+Test for binding bigint output and inout parameters
+--SKIPIF--
+<?php require('skipif_mid-refactor.inc'); ?>
+--FILE--
+<?php
+require_once("MsCommon_mid-refactor.inc");
+
+$conn = connect();
+
+// Create the table
+$tbname = "bigint_table";
+createTable($conn, $tbname, array("c1_bigint" => "bigint"));
+
+// Create a Store Procedure
+$spname = "selectBigint";
+$spSql = "CREATE PROCEDURE $spname (@c1_bigint bigint OUTPUT) AS
+          SELECT @c1_bigint = c1_bigint FROM $tbname";
+$conn->query($spSql);
+
+// Insert a large bigint
+insertRow($conn, $tbname, array("c1_bigint" => 922337203685479936));
+
+// Call store procedure with output
+$outSql = "{CALL $spname (?)}";
+$bigintOut = 0;
+$stmt = $conn->prepare($outSql);
+$stmt->bindParam(1, $bigintOut, PDO::PARAM_INT, PDO::SQLSRV_PARAM_OUT_DEFAULT_SIZE);
+$stmt->execute();
+printf("Large bigint output:\n" );
+var_dump($bigintOut);
+printf("\n");
+
+// Call store procedure with inout
+$bigintOut = 0;
+$stmt = $conn->prepare($outSql);
+$stmt->bindParam(1, $bigintOut, PDO::PARAM_INT | PDO::PARAM_INPUT_OUTPUT, PDO::SQLSRV_PARAM_OUT_DEFAULT_SIZE);
+$stmt->execute();
+printf("Large bigint inout:\n" );
+var_dump($bigintOut);
+printf("\n");
+
+$conn->exec("TRUNCATE TABLE $tbname");
+
+// Insert a small bigint
+insertRow($conn, $tbname, array("c1_bigint" => 922337203));
```

Comment:
Is there any particular reason for choosing this value?

Reply:
this is just a random value truncated from the bigint input

Reply:
perhaps add that to description?